requirements.txt

```diff
@@ -24,7 +24,7 @@ redis == 2.10.3
 yattag == 1.6.0
 xmltodict == 0.10.2
 unittest2 == 1.1.0
-influxdb == 4.1.0
+influxdb == 5.0.0
```

Comment:
Nope, all tests pass. This upgrade was needed because `influxclient.drop_measurement` was introduced in 5.0.0 (used [here](https://github.com/metabrainz/listenbrainz-server/pull/406/files#diff-6b990661c20825de5ab2cef8e5f7f19eR670))

Reply:
Famous last words! I just have a gut-punch feeling every time I see a version updated for anything as critical as a DB. I hope you're right. :)

Reply:
Understood, but it still raises my hackles and makes be really pay attention. Some level of paranoia really is useful. :)